libs/sdk-backend-mockingbird/src/recordedBackend/index.ts

```diff
@@ -182,6 +188,19 @@ function recordedWorkspace(
     };
 }
 
+function recordedDomain(domain: ObjRef): IDomain {
+    return {
+        domain,
+        securitySettings(): ISecuritySettingsService {
+            return {
+                isUrlValid(_url: string, _context: ValidationContext): Promise<boolean> {
+                    return Promise.resolve(true); // TODO what should I do with this?
```

Comment:
I think always returning true here is a good first implementation. If we ever need to mock false here, we can always add support later.

Reply:
In the end I made it configurable. User can now setup what should be returned.